cmd/go-quai/start.go

```diff
@@ -99,7 +100,25 @@ func runStart(cmd *cobra.Command, args []string) error {
 
 	if viper.IsSet(utils.MetricsEnabledFlag.Name) {
 		log.Global.Info("Starting metrics")
-		metrics_config.EnableMetrics()
+		var (
+			addr string
+			port int
+		)
+
+		if viper.IsSet(utils.MetricsHTTPFlag.Name) {
```

Comment:
It shouldn't be necessary to do an IsSet check here. If its not set then calling GetString will use the default value set in the flag automatically. Same with the next checks.